src/providers/krb5/krb5_auth.c

```diff
@@ -1023,6 +1023,9 @@ static void krb5_auth_done(struct tevent_req *subreq)
         goto done;
 
     default:
+        DEBUG(SSSDBG_IMPORTANT_INFO,
+              "The krb5_child process returned an error. Please inspect the "
+              "krb5_child.log file for more information\n");
```

Comment:
People who are not familiar with SSSD internals may not know that krb5_child.log is part of SSSD. IMO it would be good to print the full path to the log, so that users find it easier. So the line should be:
`"%s/krb5_child.log file for more information\n", LOG_PATH);`